Main/ClientApp/src/Components/ElementView.tsx

```diff
@@ -45,27 +45,38 @@ export default function ElementView(props: ElementViewProps) {
     setFaviconUrl(GetHostnameLink(url) + "/favicon.ico");
   }, [props.element]);
 
+  let onCardClick = () => {
+    let link = document.getElementById(elementUrl);
+    link?.click();
+  };
+
   return (
-    //TODO: change structure so clicking on menu doesn't redirect
-    <a
-      href={elementUrl}
-      className={classes.link}
-      target="_blank"
-      rel="noopener noreferrer"
-    >
-      <Card className={classes.root} elevation={3}>
+    <>
+      <a
+        id={elementUrl}
```

Comment:
I don't like having url as id, when someone creates to elements with the same link it may cause bugs. I think the best way would be to have the id as a new GUID and keep it as property in this component and use it to find the 'a' element

Reply:
At first i gave the same id to every element and after clicking links only one url has been activated (url of the last element), then i changed to elementUrl and as far as i tested it doesn't cause problems, becouse if you have exact same urls in different elements you will go to that url, regardles if this link comes from first or second element of the same url

Reply:
That's true, but overall it is not a good practice to keep the functionality like this, as we may need to add extra functionality to the card and forget that clicking it actually selects a different one.
In this case though, you are right and it most likely does not matter. I would add a comment for a future with a warning that the selected element may actually be a different one.